src/components/CreateResetPassword/CreateResetPassword.js

```diff
@@ -187,7 +187,7 @@ class CreateResetPassword extends Component {
                     <Field
                       id="new-password"
                       name="newPassword"
-                      autoComplete="new-password"
+                      autoComplete="on"
```

Comment:
Why did this change? `new-password` seems correct to me. Explanation from https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete: 
> When creating a new account or changing passwords, this should be used for an "Enter your new password" or "Confirm new password" field

Reply:
@zburke, thanks for explanation link, I agree with your comments, already fixed